portfolio/src/main/webapp/index.html

```diff
@@ -52,142 +48,161 @@
-                  <a id="link-to-git"
-                    href="https://github.com/nd0761/local_server"><img src="images/logo.png" alt="GitHub logo"/></a>
-                    <p>Local server for GET requests processing.</p>
-                </div>
+          <div class="column">
+            <div class="card">
+              <div class="container">
+                <a class="my-heading" id="name-of-git"><b><a>Local server</a></b></a class="my-heading">
+                <a id="link-to-git"
+                  href="https://github.com/nd0761/local_server"><img src="images/logo.png" alt="GitHub logo"/></a>
+                  <p>Local server for GET requests processing.</p>
               </div>
             </div>
+          </div>
 
-            <div class="column">
-              <div class="card">
-                <div class="container">
-                  <a class="my-heading" id="name-of-git"><b><a>Generative adversarial networks</a></b></a class="my-heading">
-                  <a id="link-to-git"
-                    href="https://github.com/Private-ML/gan"><img src="images/logo.png" alt="GitHub logo"/></a>
-                    <p>Team project on private machine learining.</p>
-                </div>
+          <div class="column">
+            <div class="card">
+              <div class="container">
+                <a class="my-heading" id="name-of-git"><b><a>Generative adversarial networks</a></b></a class="my-heading">
+                <a id="link-to-git"
+                  href="https://github.com/Private-ML/gan"><img src="images/logo.png" alt="GitHub logo"/></a>
+                  <p>Team project on private machine learining.</p>
               </div>
             </div>
-
           </div>
+
         </div>
+      </div>
 
-        <div class="hobbies"> <!-- Something about my hobbies -->
-          <p class="my-heading"><a name="hobbies"></a>Hobbies</p>
-          <p>On my free time I love watching series on Netflix and playing video games.</p>
+      <div class="hobbies"> <!-- Something about my hobbies -->
+        <p class="my-heading"><a name="hobbies"></a>Hobbies</p>
+        <p>On my free time I love watching series on Netflix and playing video games.</p>
 
-          <div class="row">
+        <div class="row">
 
-            <div class="column">
-              <div class="card">
-                <div class="container">
-                  <a><img src="images/br_ba.jpg" alt="Breaking Bad poster"/></a>
-                    <p>My most loved series. Amazing work of the actors and magnificent plot left a
-                      lasting impression.</p>
-                </div>
+          <div class="column">
+            <div class="card">
+              <div class="container">
+                <a><img src="images/br_ba.jpg" alt="Breaking Bad poster"/></a>
+                  <p>My most loved series. Amazing work of the actors and magnificent plot left a
```

Comment:
The `<p>` tag is not inside the `<a>` tag, so should probably not be indented more. Same applies to the <p> tags below.